nion/swift/ExportDialog.py

```diff
@@ -261,18 +261,18 @@ class UnitType(enum.Enum):
 
 
 class ExportSizeModel(Observable.Observable):
-    def __init__(self, display_item: DisplayItem.DisplayItem, ui: UserInterface.UserInterface) -> None:
+    def __init__(self, display_item: DisplayItem.DisplayItem, units_persistent_model: UserInterface.StringPersistentModel) -> None:
         super().__init__()
-        self.ui = ui
+        self.__units_persistent_model = units_persistent_model
         display_size = self.__calculate_display_size_in_pixels(display_item)
         self.__width = display_size.width
         self.__height = display_size.height
         self.__aspect_ratio = self.__width / self.__height
         self.__float_to_string_converter = Converter.FloatToStringConverter()
         self.__primary_field = 'width'  # Primary field to determine which text is calculater
         # Load the persisted units value, default to PIXELS if not found
```

Comment:
`# Load the persisted units value, default to PIXELS if not found`

This comment is outdated and can be removed. The default value is set by the caller.